sql/002-create-disc.sql

```diff
@@ -0,0 +1,22 @@
+use `music-db`;
+
+DROP TABLE IF EXISTS `discs`;
+
+CREATE TABLE `discs`
+(
+    `id`       int(11) NOT NULL AUTO_INCREMENT,
+    `songs`    varchar(255),
+    `artist`   varchar(255),
```

Comment:
Artist has a list of discs. Therefore, a disc should reference the id of the artist that it belongs:
`artist_id` int(11)